cmd/kubeadm/app/cmd/phases/init/addons.go

```diff
@@ -42,6 +45,12 @@ var (
 
 // NewAddonPhase returns the addon Cobra command
 func NewAddonPhase() workflow.Phase {
+	dnsLocalFlags := pflag.NewFlagSet(options.PrintManifest, pflag.ContinueOnError)
+	dnsLocalFlags.BoolVar(&dnsaddon.PrintManifestFlag, options.PrintManifest, dnsaddon.PrintManifestFlag, "Print the addon manifests to STDOUT instead of installing them")
```

Comment:
we shouldn't use a variable from another package to populate flag values.
we can store this in a local variable in addons.go - `printManifestFlag`.
then we can pass this variable to `dnsaddon.Ensure*Addon()` functions.

Reply:
Ok, I will modify it .